Competition2017/src/competition/operator_interface/OperatorCommandMap.java

```diff
@@ -27,9 +27,19 @@ public void setupMyCommands(
     public void setupCollectorCommands(
             OperatorInterface oi,
             EjectCollectorCommand eject,
-            IntakeCollectorCommand intake
-    )   {
+            IntakeCollectorCommand intake)
+    {
         oi.leftButtons.getifAvailable(1).whileHeld(eject);
         oi.leftButtons.getifAvailable(2).whileHeld(intake);
     }
+    
+    @Inject
+    public void setupAgitatorCommands(
+            OperatorInterface oi,
+            SpinAgitatorForwardsCommand forwards,
+            SpinAgitatorBackwardsCommand backwards)
+    {
+        oi.leftButtons.getifAvailable(3).whenPressed(forwards);
+        oi.leftButtons.getifAvailable(4).whenPressed(backwards);
+    }
```

Comment:
There's nothing that stops the agitator!

Either make these commands run "whileHeld" or add an additional button that's bound to a stop command.